### DISCUSSION TITLE
Welcome to ais_transponder Discussions!

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#20] [General]
## 👋 Welcome!
  I have just enabled Discussions as a place to connect with other folks, without necessarily opening tickets ("issues").
  * Ask questions.
  * Share ideas.
  * Engage.

**Comment by jgreenawalt04** [comment]
I am working on the 6.1.0 transponder.  I have all the major components.
Do you already have a BOM created for the passive components?  If not, what size are the various passive components?  Maybe there is a list in your files and I missed it.

**Reply by peterantypas** [reply]
I'll attach it here -- 6.1 is too old and I don't plan to support it going forward. I'll be posting BOMs starting with version 10.1 and later


[transponder-6.1.0_ListByValues.xlsx](https://github.com/peterantypas/maiana/files/5730543/transponder-6.1.0_ListByValues.xlsx)

**Reply by jgreenawalt04** [reply]
Thank you

**Reply by rszemeti** [reply]
V10 boards ordered from the usual place,  I think I have most of the major bits from my first build, as I ordered 5 sets of everything I think, so I should have enough to populate a V10 and no doubt have to buy a few extra bits though for the differences.

**Reply by uflex99** [reply]
Hi Peter, 
A wonderfull design you have made. About 5 years ago i had the same idea -to integrate a  SI4263/STM32F042 based AIS-receiver direct in the antenne- but I eventually discontinued it because I found the reception sensitivity unsatisfactory (compared to the results of a commercial receiver). I made another attempt with an extra LNA but that didn't improve things. I have always had the feeling that the settings of the SI4463 were far from optimal and that much better DSP processing should be possible. The starting point for my design was the MSP430 based "dAISy"-project" but the author of it experienced the same problems regardig a poor reception quality. I am very curious about your experiences on that point. 
Regards, André

![3dView1](https://user-images.githubusercontent.com/6697122/104167623-f1e34700-53fc-11eb-802a-08f3ba0d9419.jpg)
